### DISCUSSION TITLE
Very exagerated elevation gain

### DISCUSSION THREAD

**Discussion by Pilou8** [bailuk/AAT#147] [General]
Hello,

AAT is the best solution I have found for seeing and logging position, speed and heartrate when cycling.
But I still have to use Strava in parallel for only one reason: Elevation gain in AAT or when importing its gps in another app always give very exagerated elevation gains (at least twice as big as the truth).
I know this is a tough issue, but do you plan to make some improvement about it.
The simpler (and dirty) solution would be to set a coefficient to make the result generally close from the reality. Is this a realistic idea?

Thanks again for your work!

**Comment by bailuk** [comment]
Its also possible to get elevation from the phones pressure (Barometer) sensor. 
Maybe this is more precise:

1. Enable the built in barometer Sensor in `Preferences->System` from inside AAT
2. Set altitude manually once by taping `Altitude Barometer` in the Cockpit view

**Comment by bailuk** [comment]
See #118
